docs/glsl-fuzz-walkthrough.md

```diff
@@ -562,6 +562,19 @@ In the above example,
 a function body that contains a somewhat complex `pow` function call
 is enough to trigger the bug.
 
+
+## After updating a device
+
+If you update a device,
+its "fingerprint" will change
+and the server will usually reject it.
+The solution is to give the worker a new name.
+This also allows you to compare
+results across different versions of your device
+in the Web UI,
+without having to delete your old results.
```

Comment:
We should perhaps acknowledge that if you want to e.g. run reductions on already computed results, but that your worker info has changed, you can't just rename the worker.

We should perhaps provide override functionality for this.

Reply:
Perhaps, although I would argue that you would want to first check if the bugs reproduces?